apache-apisix/metadata.csv

```diff
@@ -0,0 +1,7 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+apisix.request.counter,count,,request,,Number of requests received.,0,apache_apisix,
+apisix.request.latency,gauge,,millisecond,,"The latency added by Apache APISIX, time taken to process the request.",0,apache_apisix,
+apisix.upstream.latency,gauge,,millisecond,,"The latency added by Apache APISIX, time taken since proxying the request to the upstream server till a response is received.",0,apache_apisix,
+apisix.apisix.latency,gauge,,millisecond,,"The latency added by Apache APISIX, time taken by APISIX agent to process the request.",0,apache_apisix,
+apisix.ingress.size,gauge,,byte,,Request body size in bytes.,0,apache_apisix,
+apisix.egress.size,gauge,,byte,,Response body size in bytes.,0,apache_apisix,

```

Comment:
```suggestion
metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
apisix.request.counter,count,,request,,Number of requests received.,0,apache_apisix,
apisix.request.latency,gauge,,millisecond,,"The latency added by Apache APISIX, time taken to process the request.",0,apache_apisix,
apisix.request.latency,rate,,millisecond,,"The latency added by Apache APISIX, time taken to process the request.",0,apache_apisix,
apisix.upstream.latency,gauge,,millisecond,,"The latency added by Apache APISIX, time taken since proxying the request to the upstream server till a response is received.",0,apache_apisix,
apisix.upstream.latency,rate,,millisecond,,"The latency added by Apache APISIX, time taken since proxying the request to the upstream server till a response is received.",0,apache_apisix,
apisix.apisix.latency,gauge,,millisecond,,"The latency added by Apache APISIX, time taken by APISIX agent to process the request.",0,apache_apisix,
apisix.apisix.latency,rate,,millisecond,,"The latency added by Apache APISIX, time taken by APISIX agent to process the request.",0,apache_apisix,
apisix.ingress.size,gauge,,byte,,Request body size in bytes.,0,apache_apisix,
apisix.ingress.size,rate,,byte,,Request body size in bytes.,0,apache_apisix,
apisix.egress.size,gauge,,byte,,Response body size in bytes.,0,apache_apisix,
apisix.egress.size,rate,,byte,,Response body size in bytes.,0,apache_apisix,

```